extensions/amp-image-viewer/0.1/validator-amp-image-viewer.protoascii 

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
s/2016/2018

Reply:
The validator stuff here is in very rough shape. I'll ping you for a proper review on that when this component is actually ready. =)